### DISCUSSION TITLE
Combining contigs from technical replicates

### DISCUSSION THREAD

**Discussion by ShailNair** [ablab/spades#757] [Q&A] [answered]
Hi,
I ran metaspades on my metagenomic dataset (sample with technical replicate). Further on annotation, I found a few of the genes (which I am interested in) predicted in one sample does not show up in its replicate contig. As metaspades does not support co-assembly, would it be possible to combine this two contig (from technical replicates)?

**Comment by asl** [comment]
Well, it does support – just combine all reads together. Will it work?

**Comment by ShailNair** [comment]
@asl Thought of that. but then I need to again run the assembly which takes a bit of time. I was just wondering if could combine contigs, that would have been easy.  But no complain...will try assembling by concatenating the replicates.

Thanks.

**Comment by asl** [comment, marked as answer]
Well, combining contigs is not a co-assembly. Though, for combining lots of different tools are available. Not sure about their performance though :)
